Tests/ArgumentParserUnitTests/HelpGenerationTests.swift

```diff
@@ -371,7 +371,7 @@ extension HelpGenerationTests {
     USAGE: l [--remote <remote>]
 
     OPTIONS:
-      -t, -x, -y, --remote, --when, --time, -other, --there <remote>
+      -t, -x, -y, -other, --remote, --there, --time, --when <remote>
```

Comment:
Hmmm, other than moving short names to the beginning, the old version preserved the declaration order of the names, which seems important for at least some cases. What’s motivating this change in behavior?

Reply:
that's a fair point, I thought it made more sense to put the arguments in order, but you're right, using declaration order allows the user to move depreciated flags to the back of the help, I'll back out this change.